plugins/aws/emr/emrClusterInVPC.js

```diff
@@ -0,0 +1,87 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'EMR Cluster In VPC',
+    category: 'EMR',
+    domain: 'Compute',
+    description: 'Ensure that your Amazon Elastic MapReduce (EMR) clusters are provisioned using the AWS EC2-VPC platform instead of EC2-Classic platform.',
```

Comment:
```suggestion
    description: 'Ensure that your Amazon Elastic MapReduce (EMR) clusters are provisioned using the AWS VPC platform instead of EC2-Classic platform.',
```